src/ReleaseApp/Domain/Entities/UpgradeInstructionsReleaseGroup.php

```diff
@@ -180,4 +185,12 @@ public function getJiraIssueLink(): ?string
     {
         return isset($this->body[static::JIRA_KEY]) ? $this->body[static::JIRA_KEY][static::ISSUE_LINK_KEY] : null;
     }
+
+    /**
+     * @return bool
+     */
+    public function isSecurity(): bool
+    {
+        return isset($this->body[static::SECURITY_KEY]) ? (bool)$this->body[static::SECURITY_KEY] : false;
```

Comment:
Minor: (bool) ($this->body[static::SECURITY_KEY] ?? false)

Reply:
++

Reply:
++